packages/storage-plus/src/prefix.rs

```diff
@@ -6,8 +6,9 @@ use std::marker::PhantomData;
 use cosmwasm_std::{Order, Pair, StdResult, Storage};
 use std::ops::Deref;
 
+use crate::de::Deserializable;
```

Comment:
It is confusing to mix `crate::de::Deserializable` with `serde::de::DeserializeOwned` and `serde::Serialize`.

How about another name for `crate::de::Deserializable` like `crate::de::KeyDeserializable` (or a better name)